slack-plugin/src/main/kotlin/slack/gradle/StandardProjectConfigurations.kt

```diff
@@ -694,14 +694,16 @@ internal class StandardProjectConfigurations {
         }
         // We don't set targetSdkVersion in libraries since this is controlled by the app.
 
-        // Configure generated manifest
-        // TODO disabled because it seems that disabling android resources always results in
-        //  AndroidSourceSet.manifest.srcFile to exist for some reason
-        //        AutoManifest.configure(
-        //          project = project,
-        //          libraryExtension = this@configure,
-        //          handler = slackExtension.androidHandler.libraryHandler.manifestHandler
-        //        )
+        // TODO namespace is unfortunately not a property so we can't chain this. Would be nice if
+        //  it was.
+        if (slackProperties.computeAndroidNamespace) {
+          namespace =
+            "slack" +
+              project
+                .path
+                .snakeToCamel() // handles dashes and underscores in names
```

Comment:
I'm lukewarm on this and open to just replacing dashes with dots here too since camelcase in package names isn't quite common either

Reply:
FWIW we opted for simply removing the dashes and underscores altogether in our version and everyone's been pretty happy with it. 

Here's how we thought through it with an example module we have called internal-settings:
 - `com.mycompany.internal.settings`: Doesn't make sense as the word internal in the module name means something different than what internal traditionally means in a package name.
 - `com.mycompany.internalSettings`: Like you mentioned, camel casing in a package name is odd.
 - `com.mycompany.internalsettings`: IMO the clearest option. If it comes out hard to read, maybe it's motivation to use module nesting instead of lots of dashes.

Reply:
Sold! Thanks for the detailed writeup